plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -18,26 +32,38 @@ class usdFileRelative(object):
         UI tool-tip:      kMakePathRelativeToSceneFileAnn
         option var:   mayaUsd_MakePathRelativeToSceneFile
     '''
+    # We will find (and store) some of the dialog controls so we can query them.
+    # Note: are initialized when calling the uiInit() method.
+    _fileDialog = None
+    _acceptButton = None
 
     kMakePathRelativeCheckBox = 'MakePathRelativeTo'
+    kUnresolvedPathTextField = 'UnresolvedPath'
+    kResolvedPathTextField = 'ResolvedPath'
 
-    _relativeToFilePath = None
+    # We will store whether or not the file can be made relative. Used to know when we need
+    # to update fields.
+    # Note: are initialized in the uiInit() method.
+    _canBeRelative = False
+    _haveRelativePathFields = False
 
-    @classmethod
-    def setRelativeFilePathRoot(cls, filePath):
+    @staticmethod
+    def setRelativeFilePathRoot(filePath):
         '''
         Sets to which file the relative file will be anchored.
         Set to empty or None to have the file be absolute, not relative.
         '''
-        cls._relativeToFilePath = filePath
+        global _relativeToFilePath
+        _relativeToFilePath = filePath
 
-    @classmethod
-    def getRelativeFilePathRoot(cls):
+    @staticmethod
+    def getRelativeFilePathRoot():
         '''
         Gets to which file the relative file will be anchored.
         Empty or None to have the file be absolute, not relative.
         '''
-        return cls._relativeToFilePath
+        global _relativeToFilePath
+        return _relativeToFilePath
```

Comment:
Made these static methods using global variable, to ensure we are always setting/getting the right value. There can only ever be a single instance of this dialog/class(es) at a time.